docs/api.md

```diff
@@ -0,0 +1,1683 @@
+| --------- | --------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| `cadence` | string **(required)**                   | A valid cadence script.                                                                                     |
+| `args`    | [ArgumentFunction](#`ArgumentFunction`) | Any arguments to the script if needed should be supplied via a function that returns an array of arguments. |
+| `limit`   | number                                  | Compute limit for query. :tomato: WHAT UNITS ARE THESE IN?                                                  |
+
+### Returns
+
+| Type | Description                            |
+| ---- | -------------------------------------- |
+| any  | A JSON representation of the response. |
+
+### Usage
+
+```javascript
+import * as fcl from "@onflow/fcl";
+
+const result = await fcl.query({
+  cadence: `
+    pub fun main(a: Int, b: Int, addr: Address): Int {
+      log(addr)
+      return a + b
+    }
+  `,
+  args: (arg, t) => [
+    arg(7, t.Int), // a: Int
+    arg(6, t.Int), // b: Int
+    arg("0xba1132bc08f82fe2", t.Address), // addr: Address
+  ],
+});
+console.log(result); // 13
+```
+
+### Examples
+
+- [Additional Explanation](https://gist.github.com/orodio/3bf977a0bd45b990d16fdc1459b129a2)
+
+---
+
+## `fcl.mutate({...options})`
+
+Allows you to submit transactions to the blockchain to potentially mutate the state.
+
+:warning: When being used in the browser, `fcl.mutate` uses the built-in `fcl.authz` function to produce the authorization (signatures) for the current user. When calling this method from Node.js, you will need to supply your own custom authorization function.
+
+### Options
+
+_Pass in the following as a single object with the following keys. All keys are optional unless otherwise stated._
+
+| Key        | Type                                            | Description                                                                                                                             |
+| ---------- | ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `cadence`  | string **(required)**                           | A valid cadence transaction.                                                                                                            |
+| `args`     | [ArgumentFunction](##`ArgumentFunction`)        | Any arguments to the script if needed should be supplied via a function that returns an array of arguments.                             |
+| `limit`    | number                                          | Compute limit for query. :tomato: WHAT UNITS ARE THESE IN?                                                                              |
```

Comment:
```suggestion
| `limit`    | number                                          | Compute (Gas) limit for query. Got to the [documentation](https://docs.onflow.org/flow-go-sdk/building-transactions/#gas-limit) for information about how computation cost is calculated on Flow.                                                                              |
```

Reply:
There is a spelling mistake in the suggested update: `Got to the [documentation]` -> `Go to the [documentation]`